Pollo/ViewControllers/PollsDateViewController+Extension.swift

```diff
@@ -150,20 +151,39 @@ extension PollsDateViewController: GroupControlsViewControllerDelegate {
 
 extension PollsDateViewController: SocketDelegate {
```

Comment:
The code inside this extension looks very similar to the code in the CardController extension, missing two lines of code related to the CardController's collectionView. Would it be worth factoring out 'banner' code somewhere to use it in both of these extensions and have it ready to use for any future views?

Reply:
I thought about refactoring them somewhere else but I'd have to pass around the socket/socket delegate for the reconnection calls etc. since we transfer the delegate to `CardController` from `PollsDateViewController` and both retain references to the same socket.